src/features/auth/ui/GoogleLoginButton.module.css

```
@@ -0,0 +1,108 @@
+}
+
+.gsi-material-button .gsi-material-button-content-wrapper {
+    -webkit-align-items: center;
+    align-items: center;
+    display: flex;
+    -webkit-flex-direction: row;
+    flex-direction: row;
+    -webkit-flex-wrap: nowrap;
+    flex-wrap: nowrap;
+    height: 100%;
+    /*justify-content: space-between;*/
```

Stylelint (17.9.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)